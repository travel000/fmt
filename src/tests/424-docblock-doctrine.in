<?php
//passes: PrettyPrintDocBlocks

use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity
 * @ORM\Table(name="option")
 */
class Option
{
    /**
     * @ORM\Id
     * @ORM\Column(type="integer")
     * @ORM\GeneratedValue
     */
    protected $id;
}