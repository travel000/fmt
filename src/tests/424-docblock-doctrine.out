<?php
//passes: PrettyPrintDocBlocks

use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity
 * @ORM\Table(name="option")
 */
class Option {
	/**
	 * @ORM\Id
	 * @ORM\Column(type="integer")
	 * @ORM\GeneratedValue
	 */
	protected $id;
}